openapi.yaml

```diff
@@ -248,6 +250,8 @@ paths:
           description: Bad Request
         415:
           description: Unsupported media type
+        409:
+          description: Conflict with current state
```

Comment:
A little nitpicky, but I think we should order these numerically. It seems like 404 appears before 400 a few times too — while unrelated to 409, I would suggest we swap these in this PR as well.